sdk/agrifood/azure-agrifood-farming/samples/sample_satellie_download.py

```diff
@@ -0,0 +1,168 @@
+                                [79.27248358726501, 18.041069275656195],
+                                [79.27057921886444, 18.042507660177698]
+                            ]
+                        ]
+                    )
+                )
+            )
+            print("Created")
+
+        # Queue a satellite job and wait for completion.
+        job_id = f"{job_id_prefix}-{randint(0, 1000)}"
+        print(f"Queuing satellite job {job_id}... ", end="", flush=True)
+        satellite_job_poller = client.scenes.begin_create_satellite_data_ingestion_job(
+            job_id=job_id,
+            job=SatelliteDataIngestionJob(
+                farmer_id=farmer_id,
+                boundary_id=boundary_id,
+                start_date_time=start_date_time,
+                end_date_time=end_date_time,
+                data=SatelliteData(
+                    image_names=[
+                        "LAI"
+                    ]
+                )
+            )
+        )
+        print("Queued. Waiting for completion... ", end="", flush=True)
+        satellite_job_poller.result()
+        print(f"Job completed with status {satellite_job_poller.status()}")
+
+        # Get scenes which are available in FarmBeats for our farmer and boundary of intrest.
+        print("Getting scenes list... ", end="", flush=True)
+        scenes = client.scenes.list(
+            boundary.farmer_id,
+            boundary.id,
+            start_date_time=start_date_time,
+            end_date_time=end_date_time,
+        )
+        print("Done")
+
+        # Setting up async function to parallely download images.
+        def download_image(client, file_link, root_dir):
+            file_path = parse_file_path_from_file_link(file_link)
+            out_path = Path(os.path.join(root_dir, file_path))
+            out_path.parent.mkdir(parents=True, exist_ok=True)
+            
+            print(f"Downloading image {file_path}... ", end="", flush=True)
+            with open(out_path, 'wb') as tif_file:
+                file_stream = client.scenes.download(file_path)
+                for bits in file_stream:
+                    tif_file.write(bits)
+            print("Done")
+            return str(out_path)
+
+        for scene in scenes:
+            for image_file in scene.image_files:
+                download_image(client, image_file.file_link, data_root_dir)
+
+        print("Downloads done")
+
+
+def main():
+
+    # Create client
+    try:
+        farmbeats_endpoint = os.environ['FARMBEATS_ENDPOINT']
+    except KeyError:
+        raise SystemExit(
+            "Please set the 'FARMBEATS_ENDPOINT' env variable to your FarmBeats endpoint.")
+
+    # Initialize credentials
+    credential = DefaultAzureCredential()
+
+    # Init client
+    client = FarmBeatsClient(
+        endpoint=farmbeats_endpoint,
+        credential=credential
+    )
+
+    # Run the samples
+    SatelliteSample.run(client)
+
+
+if __name__ == "__main__":
+
+    main()
```

Comment:
just as a link, once you format it more like the [sync TA samples](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_alternative_document_input.py), you can have your sample method `def sample_satellite_download` that has all of the cool farmbeats stuff, then all you need is a `if __name__ == '__main__': sample_satellite_download()` at the bottom to get it working